libdevcore/CommonData.h

```diff
@@ -394,4 +393,23 @@ std::vector<T> make_vector(Args&&... _args)
 	return result;
 }
 
+// Function that check if all characters in string are safe and return quoted parametr.
+// If there is unsafe characters it use "0x" + h256(_value, h256::AlignLeft).hex();
+std::string formatAsStringOrNumber(std::string const &_value)
+{
+	std::string resultString = "\"";
+	for (int i = 0; i < _value.length; i++)
+	{
+		if (!isalpha(_value[i]))
```

Comment:
I think `isalpha` is a bit too restrictive. Also, can it depend on the locale?

Reply:
Yea, I think you are right. What should I use there then?